### PR TITLE
chore: add to featured speakers

### PR DESCRIPTION
**Pull request by perwyl**

### CODE REVIEW

**Review by elfgoh — APPROVED** [review]
Cropping og Melvin's photo looks a bit off but non blocking. May want to improve further

File: src/sections/5.speakers.js [hunk 1]
```diff
@@ -145,10 +159,16 @@ export default () => {
                 target="speaker-tan-li-yuan-marcus-siddhant"
               />
               <SpeakerImage
-                name="Siddhant Shrivastava"
-                position="Emerging Tech Researcher at iTrust Centre for Research in Cyber Security, SUTD"
-                fixed={data.siddhant.childImageSharp.fixed}
-                target="speaker-tan-li-yuan-marcus-siddhant"
```
Comment (serverwentdown) [inline src/sections/5.speakers.js:151]: Oh no, this should be put back

**Review by serverwentdown — CHANGES_REQUESTED** [review]

File: src/sections/5.speakers.js (hunk 1 above)
Comment (perwyl) [reply src/sections/5.speakers.js:151]: putting it back. nice catch

Comment (elfgoh) [reply src/sections/5.speakers.js:151]: Wah good catch. Sleepy liao cannot see anything wahaha

**Comment by serverwentdown** [comment]
I recommend this photo: https://solveeducation.org/wp-content/uploads/melvinfix.png

**Pull request merged**